Comments

sunstoryx opened the pull request.

sunstoryx committed:
add mount for comments

sunstoryx committed:
WIP

sunstoryx committed:
add comments expose, fix image expose

sunstoryx committed:
add comments endpoints, comments entity

sunstoryx committed:
Merge remote-tracking branch 'origin/master' into comments

sunstoryx committed:
Merge remote-tracking branch 'origin/master' into comments

sunstoryx committed:
refactor comments endpoints

slenderock commented on app/api/margatsni/v1/entities/comments.rb:
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Margatsni
```
remove this file and replace with Comment entity

slenderock commented on app/api/margatsni/v1/endpoints/comments.rb:
```diff
@@ -0,0 +1,69 @@
+          end
+        end
+
+        namespace 'posts/:post_id' do
```
before { find_post }

slenderock commented on app/api/margatsni/v1/endpoints/comments.rb:
```diff
@@ -0,0 +1,69 @@
+            present :comment, comment, with: Margatsni::V1::Entities::Comment
+          end
+
+          def post
```
rename to find_post

slenderock commented on app/api/margatsni/v1/endpoints/comments.rb:
```diff
@@ -0,0 +1,69 @@
+  module V1
+    module Endpoints
+      class Comments < Margatsni::V1::BaseV1
+        helpers do
```
attr_reader :post

slenderock reviewed (changes requested):
hj

sunstoryx committed:
add expose nested comments ability to Comment entity

sunstoryx committed:
[comments]
add separated endpoint for creating nested comment
add pagination for list of comments
refactor code style

sunstoryx committed:
Merge remote-tracking branch 'origin/master' into comments

# Conflicts:
#	app/api/margatsni/v1/base_v1.rb
